tests/h/storage_test.py

```diff
@@ -184,6 +188,74 @@ def test_it_raises_when_user_is_missing_write_permission(self, pyramid_request,
 
         assert str(exc.value).startswith('group: ')
 
+    def test_it_allows_when_unscoped_group(self, pyramid_request, pyramid_config, group_service, factories, models):
```

Comment:
I wonder if `pyramid_config` is actually needed here, since the test does not use it directly?